fvm/evm/emulator/emulator.go

```diff
@@ -99,16 +105,22 @@ func (bl *BlockView) DirectCall(call *types.DirectCall) (*types.Result, error) {
 	if err != nil {
 		return nil, err
 	}
-	var res *types.Result
 	switch call.SubType {
 	case types.DepositCallSubType:
-		res, err = proc.mintTo(call.To, call.Value)
+		return proc.mintTo(call.To, call.Value)
 	case types.WithdrawCallSubType:
-		res, err = proc.withdrawFrom(call.From, call.Value)
+		return proc.withdrawFrom(call.From, call.Value)
+	case types.DeployCallSubType:
+		if call.HasNonEmptyToField() {
```

Comment:
nit: to me, this name is similar to isNaN in JS :D double positives/negatives are hard to think about
```suggestion
		if call.HasToField() {
```

Reply:
good idea, fixed it